client/src/components/Applications/AppForm/AppForm.js

```diff
@@ -138,10 +138,11 @@ export const AppSchemaForm = ({ app }) => {
       allocations: matchingHost ? state.allocations.hosts[matchingHost] : [],
       portalAlloc: state.allocations.portal_alloc,
       jobSubmission: state.jobs.submit,
-      hasDefaultAllocation: !state.allocations.loading
-        ? state.allocations.hosts[state.systems.defaultHost]
-        : true,
-      defaultHost: state.systems.defaultHost
+      hasDefaultAllocation:
+        state.allocations.loading ||
```

Comment:
Rather than assume an allocation is present, should we just disable the form if allocations are still loading?

Reply:
Ticket added: https://jira.tacc.utexas.edu/browse/FP-879